### ISSUE TITLE
[BUG] Content in edit mode is not updated...

### ISSUE THREAD

**Issue by julienwetzel** [paperless-ngx/paperless-ngx#1999]
### Description

Hello!
When I add pdf documents with an existing ocr on redo mode, it does the ocr job well but it does not update the text content up to date in edit mode. The system therefore does not search in the right content to automatically add the metatags.

### Steps to reproduce

- Put paperless in redo mode
- Upload with bad ocr pdf
- Enter in file edit mode and check content (bad ocr, not newer)

### Webserver logs

```bash
[2022-11-17 22:48:45,191] [INFO] [paperless.consumer] Consuming 2014.12.09-Romande-Energie-Acompte.pdf
[2022-11-17 22:48:45,192] [DEBUG] [paperless.consumer] Detected mime type: application/pdf
[2022-11-17 22:48:45,194] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser
[2022-11-17 22:48:45,200] [DEBUG] [paperless.consumer] Parsing 2014.12.09-Romande-Energie-Acompte.pdf...
[2022-11-17 22:48:45,523] [DEBUG] [paperless.parsing.tesseract] Extracted text from PDF file /usr/src/paperless/consume/2014.12.09-Romande-Energie-Acompte.pdf
[2022-11-17 22:48:45,654] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/usr/src/paperless/consume/2014.12.09-Romande-Energie-Acompte.pdf'), 'output_file': '/tmp/paperless/paperless-2het787x/archive.pdf', 'use_threads': True, 'jobs': '2', 'language': 'fra', 'output_type': 'pdfa', 'progress_bar': False, 'redo_ocr': True, 'clean': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': '/tmp/paperless/paperless-2het787x/sidecar.txt'}
[2022-11-17 22:49:10,447] [DEBUG] [paperless.parsing.tesseract] Using text from sidecar file
[2022-11-17 22:49:10,448] [DEBUG] [paperless.consumer] Generating thumbnail for 2014.09.26-Romande-Energie-Décompte.pdf...
[2022-11-17 22:49:10,455] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient /tmp/paperless/paperless-po_9yhfq/archive.pdf[0] /tmp/paperless/paperless-po_9yhfq/convert.webp
[2022-11-17 22:49:13,040] [DEBUG] [paperless.consumer] Saving record to database
[2022-11-17 22:49:13,042] [DEBUG] [paperless.consumer] Creation date from st_mtime: 2016-06-13 10:32:16+02:00
[2022-11-17 22:49:13,069] [INFO] [paperless.handlers] Assigning document type Facture to 2016-06-13 2014.09.26-Romande-Energie-Décompte
[2022-11-17 22:49:13,130] [DEBUG] [paperless.consumer] Deleting file /usr/src/paperless/consume/2014.09.26-Romande-Energie-Décompte.pdf
[2022-11-17 22:49:13,145] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-po_9yhfq
[2022-11-17 22:49:13,146] [INFO] [paperless.consumer] Document 2016-06-13 2014.09.26-Romande-Energie-Décompte consumption finished
```

### Paperless-ngx version

1.9.2

### Host OS

Linux 7a74269e0c63 5.15.0-52-generic #58-Ubuntu SMP Thu Oct 13 08:03:55 UTC 2022 x86_64 GNU/Linux

### Installation method

Docker - official image

### Browser

Edge, Firefox

### Other

paperless-ng to ngx migration maybe... ?

**Comment by stumpylog** [comment]
Based on that log file, OCRmyPDF was given `'redo_ocr': True` and the text was extracted from the output of OCRmyPDF.  That generally seems correct to me.  Can you explain what you mean by "bad ocr"?

**Comment by julienwetzel** [comment]
> Based on that log file, OCRmyPDF was given `'redo_ocr': True` and the text was extracted from the output of OCRmyPDF.  That generally seems correct to me.  Can you explain what you mean by "bad ocr"?

This is the original file that contains the wrong OCR

**Comment by julienwetzel** [comment]
![Screenshot_2022-11-18-12-18-07-401_org.mozilla.klar.jpg](https://user-images.githubusercontent.com/1897591/202693425-21d0be4e-c3d5-469b-bf3e-3874073b664c.jpg)

Text content in edit mode

![Screenshot_2022-11-18-12-18-33-832_org.mozilla.klar.jpg](https://user-images.githubusercontent.com/1897591/202693527-d3453c35-cf65-4983-90a9-dcf4d38214d9.jpg)

In view mode, i can copy True text.

**Comment by stumpylog** [comment]
It could be just not picked up by OCR I suppose.  Or something in ocrmypdf where not all text is found or not all text is written to the file.  Are you able to provide a sample file?  It's very difficult to debug without one

**Comment by sebmuc99** [comment]
Have same issue with PAPERLESS_OCR_MODE=redo

Only the text from that image inside the PDF file is written into the content field. the plain text is not transfered.

![image](https://user-images.githubusercontent.com/32903413/203060521-0417beac-7c2b-42ee-a9b4-536671969d6f.png)

**Comment by stumpylog** [comment]
Thank you for the sample file, that helped me uncover this edge case.

**Comment by stumpylog** [comment]
Closed in the beta with #2017
